test/distributions/bradc/ReplWithMultiDim.chpl

```diff
@@ -0,0 +1,6 @@
+use ReplicatedDist;
+
+const Space = {1..8, 1..8};
+const OneLocOnly: [0..0, 0..0] locale = Locales[0];
+const R2 = Space dmapped ReplicatedDist(targetLocales=OneLocOnly);
+writeln(R2);
```

Comment:
I just saw the error in the .good and assumed it was one of the runtime checks. I missed that it's a compile time error. No suppressif needed